### DISCUSSION TITLE
Allow merging synonymized nodes together

### DISCUSSION THREAD

**Discussion by bronwyncombs** [specify/specify7#5886] [Feature Requests]
**Reported By**
Robyn from RBGE:

> We were trying to do some tidying in the tree and are finding that we are unable to merge synonyms. One of the examples we are looking at is a synonym into a synonym (Unk. Aceraceae into Aceraceae). Is there a preference, similar to the one allowing us to create children of synonyms that would allow us to do this?

[Unk. Aceraceae tree location](https://herbrbge9323-v790.test.specifysystems.org/specify/tree/taxon/?conformation=%7E%7E1%7E5%7E35%7E305318%7E244%7E1768-%7E1512%7E35768--------)
[Aceraceae tree location](https://herbrbge9323-v790.test.specifysystems.org/specify/tree/taxon/?conformation=%7E%7E1%7E5%7E35%7E305318%7E244%7E1768-%7E1512%7E35768--------)

**Additional context**

About the mentioned pref for adding children to synonyms: 

[PR #2778](https://github.com/specify/specify7/pull/2778) | [Speciforum announcement ](https://discourse.specifysoftware.org/t/enable-creating-children-for-synonymized-nodes/987)

Related Request: https://github.com/specify/specify7/discussions/5994

**Comment by grantfitzsimmons** [comment]
> Is there a way to merge two synonyms that are duplicates? I have two occurrences of the Infraclass Acari in our Invertebrate database and each has associated specimens. Since their preferred taxa is listed as the same I’m not sure how to fix it so that there is only one.

**Reported by:** Ashley Ferguson at College of Idaho

https://discourse.specifysoftware.org/t/duplicate-synonyms-with-associated-taxa/1645?u=specify

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/duplicate-synonyms-with-associated-taxa/1645/2

**Comment by emenslin** [comment]
Possibly relevent #3292
